src/util/hass-util.html

```diff
@@ -34,6 +34,13 @@
   'input_select', 'scene', 'script', 'input_slider',
 ];
 
+window.hassUtil.FILTER_KEYS = [
```

Comment:
We should probably name it slightly better. Let's name it to what they are, instead of how they should be handled. What about `LOGIC_STATE_ATTRIBUTES` ?